src/liteclient.cc

```diff
@@ -278,6 +278,7 @@ void LiteClient::writeCurrentTarget(const Uptane::Target& t) const {
   if (!tmp.empty()) {
     ss << "CONTAINERS_SHA=\"" << tmp << "\"\n";
   }
+  ss << "PACMAN_TAG=\"" << boost::algorithm::join(tags, ",") << "\"\n";
```

Comment:
I'm not sure if subscribers will know what `PACMAN_TAG` is (vs. `DEVICE_TAG` or something similar)